File: SuperiorPackGroup/Code Classes/CustomMouseHandlerService.vb
Imports DevExpress.Services
Imports DevExpress.XtraScheduler

Public Class CustomMouseHandlerService
    Inherits MouseHandlerServiceWrapper

    Dim provider As IServiceProvider

    Public Sub New(provider As IServiceProvider, service As IMouseHandlerService)
        MyBase.New(service)

        Me.provider = provider
    End Sub

    Public Overrides Sub OnMouseWheel(e As MouseEventArgs)

        Dim scheduler As SchedulerControl = TryCast(provider, SchedulerControl)
        If scheduler IsNot Nothing AndAlso scheduler.ActiveViewType = SchedulerViewType.Timeline Then
            Return
        End If

        MyBase.OnMouseWheel(e)

    End Sub

End Class

File: SuperiorPackGroup/Code Classes/CustomMouseHandlerService.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

using DevExpress.Services;
using DevExpress.XtraScheduler;

namespace SuperiorPackGroup
{
	public class CustomMouseHandlerService : MouseHandlerServiceWrapper
	{
		private IServiceProvider provider;

		public CustomMouseHandlerService(IServiceProvider provider, IMouseHandlerService service) : base(service)
		{

			this.provider = provider;
		}

		public override void OnMouseWheel(MouseEventArgs e)
		{

			SchedulerControl scheduler = provider as SchedulerControl;
			if (scheduler != null && scheduler.ActiveViewType == SchedulerViewType.Timeline)
			{
				return;
			}

			base.OnMouseWheel(e);

		}

	}

}